package.json

```diff
@@ -19,19 +20,24 @@
     "next": "^13.4.4",
     "react": "18.2.0",
     "react-dom": "18.2.0",
+    "react-google-recaptcha": "^3.1.0",
+    "react-grid-system": "^8.1.9",
+    "react-hook-form": "^7.45.1",
     "react-query": "^3.39.3",
-    "url-join": "^5.0.0"
+    "url-join": "^5.0.0",
+    "zod": "^3.21.4"
   },
-  "peerDependencies": {    
+  "peerDependencies": {
+    "@emotion/react": "^11.11.0",
     "@emotion/styled": "^11.11.0"
   },
```

Comment:
just curious, since this app is not being imported anywhere else, why these need to be `peerDependencies`?
but maybe I'm missing something about this property 🤔

Reply:
theming from uikit works correctly because of however those caches (that you have a far better fix for in arranger I think?) work in the background (pointing to same instances I believe)

Reply:
😆 UiKit is just the gift that keeps on giving, uh?